.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,7 +14,9 @@ For new packages please briefly describe the package or provide a link to its ho
   - [ ] aarch64-linux
   - [ ] x86_64-darwin
   - [ ] aarch64-darwin
-- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
+- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
+  - [ ] `sandbox = relaxed`
+  - [ ] `sandbox = true`
```

Comment:
I think there's two other options: `false` and unset.

Reply:
Both of those are answered by not checking either option, much like it is now.

Reply:
The problem is there's no way as a reviewer to tell whether the submitter actually did anything, or even understood the question.

Reply:
There's been a lot of complaints about the list of checkmarks being too long already. I really don't think it's important. If neither's checked, assume the sandbox is off on Darwin, that's the default. The checks are useful information *some*times but not often. Having people always confirm the default setting isn't worth the effort IMO. Another PR could add more options if this one's accepted.